src/main/docker/mongo.yml

```diff
@@ -3,7 +3,7 @@ version: '2'
 services:
 
     servicenet-mongo:
-        image: mongo
+        image: mongo:3.4.19
```

Comment:
@oskarhinc why do we use the older version? The current stable version is 4.0 https://docs.mongodb.com/manual/release-notes/

Reply:
my mistake